schemas/senv_data.fbs

```diff
@@ -0,0 +1,21 @@
+//NOTE: This schema is under development.
+//If you decide to use it, be aware that backwards compatibility is not guaranteed.
+
+//Used to transmit fast sample environment data
+
+file_identifier "senv";
+
+enum Location : byte { Unknown = 0, Start, Middle, End }
+
+table senv_data {
+    Name: string (required);     // Name of the device plus channel number (e.g. "SomeName_3")
+    Channel: int;                // Channel number, currently a value from 0 to 3
+    PacketTimeStamp: ulong;      // The time stamp of the first sample in the value vector
+    TimeDelta: double;           // Time in nano seconds between samples
+    TimeStampLocation: Location; // Relevant only when oversampling has been done; is the time stamp from the start, midddle or end of the samples that were summed to produce each oversampled sample 
+    Values: [ushort] (required); // The sample values. Note that only the lower 14bits are used.
+    TimeStamps: [ulong];         // OPTIONAL, time stamps of each individual sample
+    MessageCounter: int;         // Monotonically increasing counter
```

Comment:
`ulong` or possibly `uint` might be more appropriate?
We went for `message_id: ulong` in https://github.com/ess-dmsc/streaming-data-types/blob/master/schemas/ev42_events.fbs but I prefer the name you've used here.

Reply:
Doing a quick calculation, the message counter will in the worst case scenario wrap within 20 hours when using a uint. I will change that to ulong.